lib/utils.ts

```diff
@@ -63,6 +63,8 @@ const ASYNC_UTILS = [
   'waitForDomChange',
 ];
 
+const BEFORE_HOOKS = ['beforeEach', 'beforeAll'];
```

Comment:
I think it would be better to put a more verbose name for this one, something like `TESTING_FRAMEWORK_SETUP_HOOKS` to stick to [setup/teardown concepts](https://jestjs.io/docs/en/setup-teardown) and to make clear the hooks are related to the testing framework (and not something like React hooks or anything else).